README.md

```diff
@@ -12,3 +12,26 @@ Comes with a naming system, for developerability.
 Written in Erlang. 
 
 Please see the latest [release notes](RELEASE-NOTES.md) if you want to run the software or just [follow the progress of the project in Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2124891).
+
+
+What's on board?
+===
+
+
+We keep our protocol, APIs and research spec in seprarate [protocol](https://github.com/aeternity/protocol) reporistory
+
+Restful APIs:
+[Accounts](https://github.com/aeternity/protocol/blob/master/epoch/api/account_api_usage.md)
+[Spend](https://github.com/aeternity/protocol/blob/master/epoch/api/spend_api_usage.md)
+[Oracles](https://github.com/aeternity/protocol/blob/master/epoch/api/oracle_api_usage.md)
+[Naming System](https://github.com/aeternity/protocol/blob/master/epoch/api/naming_system_api_usage.md)
+
+Websocket APIs:
+[Chain events](https://github.com/aeternity/protocol/blob/master/epoch/api/chain_ws_api.md)
```

Comment:
This looks like a duplicate of outline at https://github.com/aeternity/protocol/blob/master/epoch/api/README.md#overview
May I suggest not duplicating the links in order not to increase too much maintenance burden?